url_handlers.go

```diff
@@ -96,6 +96,33 @@ func WaitListHandler(w http.ResponseWriter, r *http.Request) {
   RenderTemplate(w, "assets/templates/waitlist.html.tmpl", nil)
 }
 
+func isAssignedBuzzerHandler(w http.ResponseWriter, r *http.Request) {
+    returnObj := map[string] interface{} {"status": false}
+
+    if !IsUserLoggedIn(GetSession(w, r)) {
+        w.WriteHeader(401)
+        returnObj["error_message"] = "Request unauthorized"
+    } elif r.Method == "POST" {
+        active_party_id = r.FormValue("active_party_id")
```

Comment:
The JSON data will be in the body, not as part of a form. So this needs to be changed I think.

`SampleJSONHandler` also shows how to do this